### PR TITLE
Navigation react to functional

### PR DESCRIPTION
**Pull request by ivanblinov2k17**

### CODE REVIEW

File: JSDemos/Demos/Drawer/LeftOrRightPosition/React/App.js [hunk 1]
```diff
@@ -1,115 +1,98 @@
...
-            {HTMLReactParser(text)}
+  return (
+    <React.Fragment>
+      <Toolbar items={toolbarItems} />
+      <Drawer
+        opened={opened}
+        openedStateMode={openedStateMode}
+        position={position}
+        revealMode={revealMode}
+        component={NavigationList}
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/Drawer/LeftOrRightPosition/React/App.js:52]: Convert navigation list to hooks

File: JSDemos/Demos/Drawer/TopOrBottomPosition/React/App.js [hunk 2]
```diff
@@ -1,113 +1,93 @@
...
-          {' '}
+  return (
+    <React.Fragment>
+      <Toolbar items={toolbarItems} />
+      <Drawer
+        opened={opened}
+        closeOnOutsideClick={onOutsideClick}
+        openedStateMode={openedStateMode}
+        position={position}
+        component={NavigationList}
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/Drawer/TopOrBottomPosition/React/App.js:52]: Convert NavigationList to hooks

File: JSDemos/Demos/MultiView/Overview/React/App.js [hunk 3]
```diff
@@ -1,82 +1,67 @@
...
+          Swipe the view horizontally to switch to the next view.
+        </i>
+      </div>
+      <MultiView
+        height={300}
+        dataSource={companies}
+        selectedIndex={selectedIndex}
+        onOptionChanged={onSelectionChanged}
+        loop={loop}
+        itemComponent={CompanyItem}
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/MultiView/Overview/React/App.js:43]: Convert CompanyItem to hooks

File: JSDemos/Demos/TabPanel/Overview/React/App.js [hunk 4]
```diff
@@ -1,103 +1,77 @@
...
+  return (
+    <div>
+      <TabPanel
+        height={260}
+        dataSource={companies}
+        selectedIndex={selectedIndex}
+        onOptionChanged={onSelectionChanged}
+        loop={loop}
+        itemTitleRender={itemTitleRender}
+        itemComponent={CompanyItem}
```
Comment (dxvladislavvolkov) [inline JSDemos/Demos/TabPanel/Overview/React/App.js:42]: CompanyItem to hooks

**Review by dxvladislavvolkov — APPROVED** [review]

**Review by alexanderPolosatov — APPROVED** [review]

**Pull request merged**